realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -1322,6 +1378,8 @@ void BaseRealSenseNode::setupDevice()
 void BaseRealSenseNode::setupPublishers()
 {
     ROS_INFO("setupPublishers...");
+    // Kiwi - Publish camera pitch
+    _cam_pitch_publisher = _node.create_publisher<std_msgs::msg::Float32>("/camera/pitch", rclcpp::QoS(1).keep_all().transient_local().reliable());
```

Comment:
Don't use absolute namespace for topics, this node uses its own namespace so it will be prepended to the topic